public/plugin/file-service/entry.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 16 16"><defs/><path stroke="#222326" d="M8 14.8A6.8 6.8 0 108 1.2a6.8 6.8 0 000 13.6z"/><path fill="#222326" d="M9.3 10.4l-.1-.2V10a1.8 1.8 0 01-1 .6H6.8l-.6-.4A1.5 1.5 0 015.8 9c0-.6.2-1 .6-1.3.4-.4 1-.5 2-.5H9v-.3c0-.3-.1-.5-.3-.6-.1-.2-.4-.3-.7-.3l-.6.2-.2.5H5.9a1.5 1.5 0 01.6-1.2c.2-.1.4-.3.7-.3a3.3 3.3 0 011.9 0c.2 0 .5.1.7.3.2.1.4.3.5.6l.1.8v2.3l.1.7c0 .2 0 .3.2.5H9.3zM8 9.4a1.4 1.4 0 001-.3V8h-.6a2 2 0 00-.6 0 1 1 0 00-.3.2l-.3.3v.3c0 .2 0 .4.2.5l.6.2z"/></svg>
```

Comment:
Move to a tsx file

Reply:
Wait a new entry icon for @OiCkilL 
CC @neruthes